app/templates/my-masterclasses.html

```diff
@@ -33,7 +33,10 @@ <h2 class="govuk-heading-m"><a class="govuk-link--no-visited-state" href="{{ url
                 </tr>
             </table>
         </div>
-        <hr> {% endfor %}
+        <hr>
+
+        {% endfor %}
+
```

Comment:
Could you combine this with the following commit? Seems odd and extraneous

Reply:
Agreed